x-pack/plugins/security_solution/server/lib/detection_engine/routes/index/delete_index_route.ts

```diff
@@ -63,9 +62,13 @@ export const deleteIndexRoute = (router: SecuritySolutionPluginRouter) => {
           if (policyExists) {
             await deletePolicy(esClient, index);
           }
-          const templateExists = await getTemplateExists(esClient, index);
+          const templateExists = await esClient.indices.existsIndexTemplate({ name: index });
           if (templateExists) {
-            await deleteTemplate(esClient, index);
```

Comment:
Similar question here: are `getTemplateExists` and `deleteTemplate` now deprecated and shouldn't be used (so should be deleted probably)?
Or maybe can we update them with the new implementation or rename to `getLegacyTemplateExists` and add a new one `getNewTemplateExists`?

Reply:
I think they're unused now. Do you think it's worth using functions like that as simple wrappers of ES API calls? Without having any extra logic in the functions it seems like they're extra overhead without much benefit.